src/vos/vos_aggregate.c

```diff
@@ -209,15 +210,43 @@ reset_agg_pos(vos_iter_type_t type, struct vos_agg_param *agg_param)
 	}
 }
 
+static inline bool
+need_aggregate(struct vos_agg_param *agg_param, vos_iter_entry_t *entry)
+{
+	struct vos_container	*cont = vos_hdl2cont(agg_param->ap_coh);
+
+	D_DEBUG(DB_EPC, "check_ts:%d, hae:"DF_U64", last_update:"DF_U64", "
+		"flags:%u\n", agg_param->ap_check_ts,
+		cont->vc_cont_df->cd_hae, entry->ie_last_update,
+		entry->ie_vis_flags);
+
+	/* Don't skip aggregation when the aggregate EPR is loer than HAE. */
```

Comment:
*lower

Reply:
oops, will fix it if the patch needs be refreshed, otherwise, will fix it in later patches.